.config-schema.json

```diff
@@ -310,6 +310,12 @@
                             "type": "string",
                             "default": "0.0.0.0:4317",
                             "x-env-variable": "OPENFGA_TRACE_OTLP_ENDPOINT"
+                        },
+                        "insecure": {
+                            "description": "Whether to use insecure connectio for the trace collector",
+                            "type": "boolean",
+                            "default": false,
+                            "x-env-variable": "OPENFGA_TRACE_OTLP_INSECURE"
```

Comment:
Please update `cmd/run/flags.go` to include this new flag, otherwise if you change `image: openfga/openfga:latest` to `build: .` in the `docker-compose.yaml` file and then run `docker-compose up`, it panics on startup

Reply:
Done.